modules/spo-recording-profiles.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-recording-profiles_{context}"]
+= Recording profiles from workloads
+
+The Security Profiles Operator can record {type} profiles with `ProfileRecording` objects.
```

Comment:
My understanding of the `ProfileRecording` object was that it recorded syscalls, which really helps in creating the profile.

One minor suggestion would be to say:

```
... can record system calls with `ProfileRecording` objects, making it easier to create baseline profiles for applications.
```